src/modules/Tab/TabPane.jsx

```diff
@@ -0,0 +1,71 @@
+import Tab from './Tab';
+
+export default {
+  name: 'SuiTabPane',
+  meta: {
+    parent: 'SuiTab',
+  },
+  props: {
+    label: {
+      type: String,
+      required: true,
+    },
+    attached: {
+      type: [Boolean, String],
+      default: true,
+    },
+    loading: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  data: () => ({
+    active: false,
+  }),
+  computed: {
+    classList() {
+      const list = [
+        'ui',
+        'tab',
+        'segment',
+        { loading: this.active && this.loading },
+        { attached: this.attached },
+        { active: this.active },
+      ];
+
+      if (typeof this.attached === 'string') {
+        list.push(this.attached);
+      }
+
+      return list;
+    },
+  },
+  mounted() {
+    this.findParent().addTab(this);
+  },
+  methods: {
+    findParent() {
+      let parent = this.$parent;
+      const name = Tab.name;
+
+      while (parent.$options.name !== name) {
+        parent = parent.$parent;
+      }
+
+      return parent;
```

Comment:
I'd throw an exception if `TabPane` isn't inside a `Tab`

Reply:
`findParent` shouldn't throw anything because it's just a helper function to find a `Tab` component

Reply:
What I'm saying is that if someone doesn't put TabPane under Tab, the error will probably be something like `Cannot find addTab of undefined`. I'd prefer a more descriptive message

Reply:
Yeah, right. Will do

Reply:
> I'd throw an exception if `TabPane` isn't inside a `Tab`

I get this error while creating dynamic tabs using v-for. Any work around for this?